x-pack/plugins/security_solution/public/flyout/entity_details/user_right/index.tsx

```diff
@@ -83,9 +83,7 @@ export const UserPanel = ({ contextID, scopeId, userName, isDraggable }: UserPan
       openLeftPanel({
         id: UserDetailsPanelKey,
         params: {
-          riskInputs: {
-            alertIds: userRiskData?.user.risk.inputs?.map(({ id }) => id) ?? [],
-          },
+          riskScore: userRiskData,
```

Comment:
We shouldn't do this.
The expandable flyout state syncs with the URL, meaning the `userRiskData` object is stringified and stored inside the URL. Because it is a complex object, it will create a long URL. It also allows users to update the URL, possibly creating some unexpected scenarios manually.

My suggestion is to update the risk input panel and make it fetch the `userRiskData id since if we fetch it by entity name, we can't guarantee that it will return the same data.

Reply:
yup @machadoum is bringing a valid point here. You should think of this params property as a way to pass the bare minimum ans smallest amount of data possible for the panel to fetch it's own data.
This will guarantee that when a users shares a url with the flyout open to someone else, it will small and the data will be loaded correctly

Reply:
Sure, I will change it to fetch data, when we open risk inputs